### ISSUE TITLE
Option to display the full list of enabled linters

### ISSUE THREAD

**Issue by peng4smart** [golangci/golangci-lint#2824]
### Your feature request related to a problem? Please describe.

Since there are no messages when a linter doesn't catch anything, there is no way to know if a linter has been run against the code.

That is when there is no message, there is no way to tell that's because no catches or no linters.

Therefore, please consider adding an option to list all linters to be enabled first or all inters have been run after. This could be either a new command/config option, or just additional info to be output when -v or --verbose is enabled.

### Describe the solution you'd like.

The full list of linters can be printed before or after all linter checks. It can be triggered by either a new option or the existing -v/--verbose option.

### Describe alternatives you've considered.

Alternatively, it may print the linter name always even if the linter doesn't find anything.

**Comment by bombsimon** [comment]
The `-v` flag already does this, doesn't it?

```sh
golangci-lint run --no-config --enable-all -v
INFO [lintersdb] Active 88 linters: [asciicheck bidichk bodyclose containedctx contextcheck cyclop deadcode decorder depguard dogsled dupl durationcheck errcheck errchkjson errname errorlint exhaustive exhaustivestruct exportloopref forbidigo forcetypeassert funlen gci gochecknoglobals gochecknoinits gocognit goconst gocritic gocyclo godot godox goerr113 gofmt gofumpt goheader goimports golint gomnd gomoddirectives gomodguard goprintffuncname gosec gosimple govet grouper ifshort importas ineffassign interfacer ireturn lll maintidx makezero maligned misspell nakedret nestif nilerr nilnil nlreturn noctx nolintlint paralleltest prealloc predeclared promlinter revive rowserrcheck scopelint sqlclosecheck staticcheck structcheck stylecheck tagliatelle tenv testpackage thelper tparallel typecheck unconvert unparam unused varcheck varnamelen wastedassign whitespace wrapcheck wsl]
```

```sh
golangci-lint run --no-config --disable-all -E gomodguard,unparam,errcheck -v
INFO [lintersdb] Active 3 linters: [errcheck gomodguard unparam]
```

**Comment by peng4smart** [comment]
Sorry, my bad.

I saw the "Active 66 linters" and "Active presets" in my output. I noticed the "Active presets" is followed by the list of presets. However, for some reason, I did not realize following "Active 66 linters" is the list of linters enabled.

I'll close this request. Thank you very much for your instant response!
